test/unit_tests/test_example_client.py

```diff
@@ -0,0 +1,70 @@
+import torch
+import torchvision.models as models
+from pytorchfi.core import fault_injection as pfi_core
```

Comment:
I think we don't need to rename to `pfi_core`, maybe just keep as `fault_injection` and use that since its used once and tends to be more readable